roles/StackStorm.st2/tasks/auth-rbac.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Create directory to store roles and assignments
+  file:
+    path: "/opt/stackstorm/rbac/{{ item }}"
+    recurse: yes
+    state: directory
+  loop:
+    - roles
+    - assignments
+  when: st2_rbac_enable|bool
+
+- name: Copy defined RBAC roles to /opt/stackstorm/rbac/roles directory
+  become: yes 
+  template:
+    src: rbac_roles/roles.yml.j2
+    dest: /opt/stackstorm/rbac/roles/{{ item.name }}.yaml
+    owner: st2
+    group: st2
+  loop: "{{ st2_rbac.roles }}"
+  when: st2_rbac_enable|bool and st2_rbac.roles is defined
+  notify: restart st2auth
+
+- name: Copy RBAC assignments to /opt/stackstorm/rbac/assignments directory
+  become: yes 
+  template:
+    src: rbac_assignments/assignments.yml.j2
+    dest: /opt/stackstorm/rbac/assignments/{{ item.name }}.yaml
+    owner: st2
+    group: st2
+  loop: "{{ st2_rbac.assignments }}"
+  when: st2_rbac_enable|bool
+  notify: restart st2auth
+
+- name: Enable RBAC in st2 configuration
+  become: yes
+  ini_file:
+    dest: /etc/st2/st2.conf
+    section: rbac
+    option: enable
+    value: True
+    backup: yes
+  when: st2_rbac_enable|bool
+  notify:
+    - restart st2api
+    - restart st2auth
+
+- name: Disable RBAC in st2 configuration
+  become: yes
+  ini_file:
+    dest: /etc/st2/st2.conf
+    section: rbac
+    option: enable
+    value: False
+    backup: yes
+  when: not st2_rbac_enable|bool
+  notify:
+    - restart st2api
+    - restart st2auth
```

Comment:
Can you please include a new line at the end of the file?
Ansible-lint has more opinions about other minor things like this.